### DISCUSSION TITLE
[Solarized-light] Right-Panel Selected Tab Colour Just Seems... Off.

### DISCUSSION THREAD

**Discussion by My-Random-Thoughts** [node-red-contrib-themes/theme-collection#472] [General] [answered]
I have been using the very old Solarized-Light theme that had provisions for scollbars for quite a while (very old).
When I upgraded to the current version the select tab on the right hand panel just seemed off.

I have been playing and think that changing all instances of #dfca88 to #d9ceaf would be a better fit.
Using the tab background colour (#fdf6e3) I used https://paletton.com/#uid=1170u0k3ivK05X11cID6irhaln9 to get a better match.

Current:
![dfca88](https://github.com/user-attachments/assets/789920a3-24fa-48d5-9aae-b71b68347831)

New:
![d9ceaf](https://github.com/user-attachments/assets/e4694dae-c848-4448-acd2-513151a505cd)

I think the new colour fits more with the theme.

**Comment by bonanitech** [comment]
Hey @My-Random-Thoughts, thanks for reporting this.

I agree with you. That should not have been changed.

**Comment by bonanitech** [comment, marked as answer]
I just released [4.0.5](https://github.com/node-red-contrib-themes/theme-collection/releases/tag/4.0.5) with the fix.
